cpp/test/unit_tests.cpp

```diff
@@ -584,6 +584,7 @@ TEST_CASE("Message index records", "[writer]") {
   REQUIRE(messageIndexChannelIds[1] == channel2.id);
 }
 
+#ifndef MCAP_COMPRESSION_NO_LZ4
```

Comment:
is there not a corresponding test case for zstd?

Reply:
There is not.